### DISCUSSION TITLE
Additions + Suggestions (Not Interested Function + Recommendation Section)

### DISCUSSION THREAD

**Discussion by twinkdoesdrag** [FreeTubeApp/FreeTube#4444] [General]
First, love the app and everyone who works on it thank you so much for saving me from YouTube's ad-pocolypse!
Second, please, please, please add a "not interested" function when you hover your mouse over a video/click on the three dots and please add in a recommended section so I'm not exclusively seeing content from people I've subscribed to. Thank you so much!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Your requests are already documented in #677 and https://github.com/FreeTubeApp/FreeTube/issues/1362

They will be addressed at some point but they are a low priority at the moment.
